.github/actions/build-push-image/action.yaml

```diff
@@ -118,4 +119,5 @@ runs:
         tags: quay.io/${{ inputs.repository }}:${{ inputs.version }}
         cache-from: type=gha
         cache-to: type=gha,mode=max
+        sbom: true
```

Comment:
Hopefully this is an easy win for including SBOM in our images.

https://docs.docker.com/engine/reference/commandline/buildx_build/#attest